### PR TITLE
add adc samples and fix scorecards

### PR DESCRIPTION
**Pull request by jimmy121192**
1. Add adc sample flow
2. Fix scorecards of the package on node-red

### CODE REVIEW

**Review by maria-mz — DISMISSED** [review]

File: examples/ADC Flow Sample.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        "y": 140,
+        "wires": []
+    },
+    {
+        "id": "3a5834a3ade508b4",
+        "type": "adc",
+        "z": "aa7eb918109cfafe",
+        "name": "",
+        "transport": "Local",
+        "tcpAddress": "192.168.1.193",
```
Comment (farzadpanahi) [inline examples/ADC Flow Sample.json]: do we need tcp address value?

File: examples/ADC and DAC Connection Sample.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            ]
+        ]
+    },
+    {
+        "id": "a3d6633ce11741b6",
+        "type": "adc",
+        "z": "1bda2585017050e3",
+        "name": "",
+        "transport": "Local",
+        "tcpAddress": "192.168.1.193",
```
Comment (farzadpanahi) [inline examples/ADC and DAC Connection Sample.json]: same here

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

**Review by naomiven — DISMISSED** [review]

File: examples/ADC Flow Sample.json (hunk 1 above)
Comment (jimmy121192) [reply examples/ADC Flow Sample.json]: The default transport is Local, so the tcpAddress wouldn't show up. I left the IP there so the user knows what to input. 
I have removed it.

File: examples/ADC and DAC Connection Sample.json (hunk 2 above)
Comment (jimmy121192) [reply examples/ADC and DAC Connection Sample.json]: Updated

**Review by farzadpanahi — DISMISSED** [review]

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**